mod/ping.php

```diff
@@ -337,28 +337,16 @@ function ping_init(App $a)
 			}
 			return ($adate < $bdate) ? 1 : -1;
 		};
-		usort($notifs, $sort_function);
-
-		if (DBA::isResult($notifs)) {
-			foreach ($notifs as $notif) {
-				$contact = Contact::getByURL($notif['url'], false, ['micro', 'id', 'avatar']);
-				$notif['photo'] = Contact::getMicro($contact, $notif['photo']);
-
-				$local_time = DateTimeFormat::local($notif['date']);
-
-				$notifications[] = [
```

Comment:
Why is this removed?

Reply:
It's been replaced by a much simpler `array_walk` call since only two keys of the array items have to be updated.

Reply:
Well, some PHP is still black magic to me.